ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -1639,6 +1642,10 @@ export class OrMwcInput extends LitElement {
                 (searchableElement as HTMLInputElement).value = "";
             }
         }
+
+        if (this.type === InputType.CHECKBOX_LIST) {
+            this.dispatchEvent(new OrInputChangedEvent(newValue.join(','), previousValue, enterPressed));
```

Comment:
What's the reason to flatten the array into a comma separated string? Where ever the value event is consumed should expect an array value if the input is multiselect.

Reply:
Not necessary anymore. In one of my try I moved to that and forget to clean it. It's updated now